docs/user-guide/src/version_support.md

```diff
@@ -39,37 +41,36 @@ The compatability of IPAM and CAPM3 API versions with CAPI is discussed
 ## Baremetal Operator
 
 Since `capm3-v1.1.2`, BMO follows the semantic versioning scheme for its own
-release cycle, the same way as CAPM3 and IPAM. Currently, we have release-0.4
-and release-0.3 release branches for `v0.4.x` and `v0.3.x` release cycle
-respectively and as such these two braches are maintained as supported releases.
+release cycle, the same way as CAPM3 and IPAM. Currently, we have release-0.5, release-0.4
+and release-0.3 release branches for `v0.5.x` `v0.4.x` and `v0.3.x` release cycle
+respectively and as such two braches are maintained as supported releases.
 Following table summarizes BMO release/test process:
 
 | Minor release | Status    |
 | ------------- | --------- |
+| v0.5          | Supported |
 | v0.4          | Supported |
 | v0.3          | Tested    |
-| v0.2          | Tested    |
+| v0.2          | EOL       |
 | v0.1          | EOL       |
 
 ## Image tags
 
 The Metal³ team provides container images for all the main projects and also
 many auxilary tools needed for tests or otherwise useful. Some of these images
 are tagged in a way that makes it easy to identify what version of Cluster API
-provider Metal³ they are tested with. For example, we tag Ironic and MariaDB
-container images with tags like `capm3-v1.4.0`, where `v1.4.0` would be the
-CAPM3 release it was tested with. Prior to CAPM3 release v1.1.3 this also
-applied to the Baremetal Operator.
+provider Metal³ they are tested with. For example, we tag MariaDB
+container images with tags like `capm3-v1.6.0`, where `v1.6.0` would be the
+CAPM3 release it was tested with.
 
 All container images are published through the
 [Metal³ organization in Quay](https://quay.io/organization/metal3-io).
 Here are some examples:
 
-- quay.io/metal3-io/cluster-api-provider-metal3:v1.5.0
-- quay.io/metal3-io/baremetal-operator:v0.4.0
-- quay.io/metal3-io/ip-address-manager:v1.5.0
-- quay.io/metal3-io/ironic:capm3-v1.5.0
```

Comment:
Why are you removing Ironic here?

Reply:
There is no ironic tag release for v1.6.0, thats why I removed it

Reply:
I am adding the irornic tag, thanks for the catch